apps/docs/src/app/platform/component-docs/platform-forms/platform-datetime-picker/platform-datetime-picker-examples/platform-datetime-picker-reactive-example.component.ts

```diff
@@ -28,7 +28,7 @@ export class PlatformDatetimePickerReactiveExampleComponent {
 
     onSubmit(): void {
         if (this.datetimePickerForm.valid) {
-            alert('Form Value: ' + this.datetimePickerForm.value);
+            alert('Form Value: ' + JSON.stringify(this.datetimePickerForm.value));
```

Comment:
If we take into concideration other components, they usualy print form or control value into some paragraph. Maybe this should be done in the same way for the sake of consistency?

Reply:
Ok. Maybe I can find somewhere an example?

Reply:
https://fundamental-ngx.netlify.app/#/core/checkbox#checkbox-reactive-forms-example

Reply:
Thanks! I'll fix it